doc/source/amazon-rds.rst

```diff
@@ -38,7 +38,8 @@ on Amazon RDS:
 .. code-block:: bash
```

Comment:
`--create-user` will not work with RDS anymore as we now have a single `pmm` user now with combined privileges including SUPER which is not possible with RDS.
So we may say to create user manually and then provide credentials.

`--create-user` privileges now are:
```
SELECT, PROCESS, REPLICATION CLIENT, SUPER ON *.* TO 'pmm'@'%' IDENTIFIED BY '...' WITH MAX_USER_CONNECTIONS 5;
GRANT UPDATE, DELETE, DROP ON `performance_schema`.* TO 'pmm'@'%';
```